src/controllers/showController.ts

```diff
@@ -28,13 +42,43 @@ export function getAllShows(): any {}
  * @param {*} attributes
  * @returns {(any | undefined)}
  */
-export function updateShow(id: number, attributes: any): any | undefined {}
+export async function updateShow(
+  id: number,
+  attributes: ShowAttributes
+): Promise<ShowInstance | null> {
+  const [numberOfUpdatedShows, updatedShows] = await Show.update(
+    attributes,
+    {
+      where: { id },
+      returning: true,
+    }
+  );
+  if (numberOfUpdatedShows === 1) {
+    return updatedShows[0];
+} else if (numberOfUpdatedShows !== 0) {
+    // throw new
+    throw new Error(
+    'More than 1 rows updated from single id in `updateShow`! This is bad!'
+);
```

Comment:
formatting seems a little wonky here make sure you run prettier through your editor? Maybe this is just github tho